packages/react-internal/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts

```diff
@@ -344,6 +344,7 @@ export const getStyles = (props: IChoiceGroupOptionStyleProps): IChoiceGroupOpti
       },
       checked && {
         borderColor: iconOrImageChoiceBorderCheckedColor,
+        border: `1px solid ${focusBorderColor}`,
```

Comment:
I don't think we should have a border visible by default--I can't see design being okay with that. Typically we only show borders on keyboard focus using `getFocusBorder`.